### DISCUSSION TITLE
Failing e2e

### DISCUSSION THREAD

**Discussion by sztadii** [vercel/vercel#5512] [General]
Hi Vercel team.
Before I will talk about the issue I would like to say I love your product, cause with your tool I configure the whole CI / CD pipeline with just a few clicks ( usually I spend much more on that ).

The issue/recommendation.
I am worried about the quality of the new features when I see a failing build.
I understand just one e2e test is not passing and many of them are working.
So if you do not have time for fixing it I would recommend temporary skip just that test, because in this way you will do not know if during the development you are not adding new issues, cause you will think that only that test is failing and you will not see that other will start failing too.

What do you think about my recommendation?

**Comment by mcsdevv** [comment]
Hello @sztadii, can you tell us what tests you are referring to?

**Comment by sztadii** [comment]
I was referring to it

https://github.com/vercel/vercel/runs/1449077955

![Screenshot 2020-12-04 at 18 22 41](https://user-images.githubusercontent.com/29600182/101174832-d21e6d80-365d-11eb-99e7-7a00da3dc3fa.png)

So as a user when I see so many failing commits on the master branch I am worried that it is ready for my production projects.

**Comment by sztadii** [comment]
Your product is great, but when potential clients like me want to invest time in building future products on your platform I want to be sure everything on the master will be stable. But when I see only this

![Screenshot 2020-12-04 at 18 27 04](https://user-images.githubusercontent.com/29600182/101175441-9d5ee600-365e-11eb-9586-ae26354e506c.png)

I am worried about stability

**Comment by ijjk** [comment]
Hi, the above mentioned tests are being updated to test the features more consistently [here](https://github.com/vercel/vercel/pull/5526). Note, some tests do fail from other factors like the GitHub Actions environment, npm, etc encountering problems which we try to account for as much as we can.
